packages/app/src/Pages/settings/Relays.tsx

```diff
@@ -41,29 +45,72 @@ const RelaySettingsPage = () => {
           <input
             type="text"
             className="f-grow"
-            placeholder="wss://my-relay.com"
+            placeholder="wss://relay.com wss://relay1.com;wss://relay2.com,wss://relay3.com"
             value={newRelay}
             onChange={e => setNewRelay(e.target.value)}
           />
         </div>
-        <button className="secondary mb10" onClick={() => addNewRelay()}>
-          <FormattedMessage {...messages.Add} />
-        </button>
+        <small className="mb10">
+          <FormattedMessage {...messages.AddRelaysHelp} />
+        </small>
+        <div className="flex">
+          <div className="f-grow"></div>
+          <button className="secondary mb10" onClick={() => addNewRelay()}>
+            <FormattedMessage {...messages.AddRelays} />
+          </button>
+        </div>
       </>
     );
   }
 
   function addNewRelay() {
-    if ((newRelay?.length ?? 0) > 0) {
-      const parsed = new URL(newRelay ?? "");
+    if (newRelay?.length) {
+      let seps = "; ,".split("");
```

Comment:
Doesn't seem necessary to support multiple separators, space alone seems fine

Reply:
I prefer to use this list of characters for delimited lists